### PR TITLE
Add new FAQs

### PR DESCRIPTION
**Pull request by TucksonDev**
[Fess on classic transactions](https://nitro-docs-git-add-new-faqs-offchain-labs.vercel.app/for-devs/troubleshooting-building#why-is-the-fee-of-some-classic-transactions-slightly-different-than-the-multiplication-of-gaslimit-and-effectivegasprice)

[Increase contract size limit on Orbit](https://nitro-docs-git-add-new-faqs-offchain-labs.vercel.app/launch-orbit-chain/troubleshooting-building-orbit#can-i-increase-the-maximum-contract-size-for-my-orbit-chain)

[Withdraw funds from Arbitrum (without Sequencer)](https://nitro-docs-git-add-new-faqs-offchain-labs.vercel.app/learn-more/faq#can-i-withdraw-my-funds-from-arbitrum-back-to-ethereum-without-going-through-the-sequencer-what-about-funds-that-are-in-a-contract)

[Incorrect sender on classic retryables](https://nitro-docs-git-add-new-faqs-offchain-labs.vercel.app/node-running/faq#when-querying-classic-transactions-from-a-nitro-node-i-sometimes-get-incorrect-data-like-the-zero-address-as-the-sender-why-is-that)

### CODE REVIEW

File: website/static/get-started-faqs.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
...
+{"question": "Can I withdraw my funds from Arbitrum back to Ethereum without going through the sequencer? What about funds that are in a contract?","answer": "<p>Yes, it is possible to permissionlesly send a message from Ethereum to be executed on Arbitrum, bypassing the sequencer. You can do this by using the DelayedInbox contract and force-including the message after a certain amount of time has passed (currently ~24 hours). You can find more information about this behavior <a href=\"https://docs.arbitrum.io/sequencer#unhappyuncommon-case-sequencer-isnt-doing-its-job\">here</a>.</p>\n\n<p>Keep in mind that you can execute any message in this way, be it a withdrawal of funds back to Ethereum, or a call to a contract.</p>\n\n<p>You can also find an example of force-inclusion in <a href=\"https://github.com/OffchainLabs/arbitrum-tutorials/tree/master/packages/delayedInbox-l2msg\">this tutorial</a>. </p>\n\n<p></p>\n\n","key": "can-i-withdraw-my-funds-from-arbitrum-back-to-ethereum-without-going-through-the-sequencer-what-about-funds-that-are-in-a-contract"},
```
Comment (mahsamoosavi) [inline website/static/get-started-faqs.json:20]:
```suggestion
{"question": "Can I withdraw my funds from Arbitrum back to Ethereum without going through the Sequencer? What about funds that are in a contract?","answer": "<p>Yes, it is possible to permissionlessly send a message from Ethereum to be executed on Arbitrum, and bypassing the Sequencer. You can do this by using the DelayedInbox contract and force-including the message after a certain amount of time has passed (currently ~24 hours). You can find more information about this behavior <a href=\"https://docs.arbitrum.io/sequencer#unhappyuncommon-case-sequencer-isnt-doing-its-job\">here</a>.</p>\n\n<p>Keep in mind that you can execute any message in this way, be it a withdrawal of funds back to Ethereum, or a call to a contract.</p>\n\n<p>You can also find an example of force-inclusion in <a href=\"https://github.com/OffchainLabs/arbitrum-tutorials/tree/master/packages/delayedInbox-l2msg\">this tutorial</a>. </p>\n\n<p></p>\n\n","key": "can-i-withdraw-my-funds-from-arbitrum-back-to-ethereum-without-going-through-the-sequencer-what-about-funds-that-are-in-a-contract"},
```

File: website/static/node-running-faqs.json [hunk 2]
```diff
@@ -11,5 +11,6 @@
...
+{"question": "When querying classic transactions from a Nitro node, I sometimes get incorrect data, like the zero address as the sender. Why is that?","answer": "<p>Some old Nitro genesis database snapshots didn't properly set the retry sender for classic blocks and contain said error. If you need to access that information, you can either resync your nitro node with one of the <a href=\"https://snapshot.arbitrum.foundation/index.html\">current snapshots</a>, or <a href=\"https://docs.arbitrum.io/node-running/how-tos/running-a-classic-node\">run a classic node</a> along with your nitro node and configure a redirection for requests to classic blocks. Please note that this only happens on Arbitrum One.</p>\n\n<p></p>\n\n","key": "when-querying-classic-transactions-from-a-nitro-node-i-sometimes-get-incorrect-data-like-the-zero-address-as-the-sender-why-is-that"}
```
Comment (mahsamoosavi) [inline website/static/node-running-faqs.json:15]:
```suggestion
{"question": "When querying Classic transactions from a Nitro node, I sometimes get incorrect data, like the zero address as the sender. Why is that?","answer": "<p>Some old Nitro genesis database snapshots didn't properly set the retry sender for Classic blocks and contain said error. If you need to access that information, you can either resync your nitro node with one of the <a href=\"https://snapshot.arbitrum.foundation/index.html\">current snapshots</a>, or <a href=\"https://docs.arbitrum.io/node-running/how-tos/running-a-classic-node\">run a Classic node</a> along with your nitro node and configure a redirection for requests to classic blocks. Please note that this only happens on Arbitrum One.</p>\n\n<p></p>\n\n","key": "when-querying-classic-transactions-from-a-nitro-node-i-sometimes-get-incorrect-data-like-the-zero-address-as-the-sender-why-is-that"}
```

**Review by mahsamoosavi — CHANGES_REQUESTED** [review]

File: website/static/get-started-faqs.json [hunk 3]
```diff
@@ -17,7 +17,8 @@
...
+{"question": "Can I withdraw my funds from Arbitrum back to Ethereum without going through the sequencer? What about funds that are in a contract?","answer": "<p>Yes, it is possible to permissionlessly send a message from Ethereum to be executed on Arbitrum, and bypassing the Sequencer. You can do this by using the DelayedInbox contract and force-including the message after a certain amount of time has passed (currently ~24 hours). You can find more information about this behavior <a href=\"https://docs.arbitrum.io/sequencer#unhappyuncommon-case-sequencer-isnt-doing-its-job\">here</a>.</p>\n\n<p>Keep in mind that you can execute any message in this way, be it a withdrawal of funds back to Ethereum, or a call to a contract.</p>\n\n<p>You can also find an example of force-inclusion in <a href=\"https://github.com/OffchainLabs/arbitrum-tutorials/tree/master/packages/delayedInbox-l2msg\">this tutorial</a>. </p>\n\n<p></p>\n\n","key": "can-i-withdraw-my-funds-from-arbitrum-back-to-ethereum-without-going-through-the-sequencer-what-about-funds-that-are-in-a-contract"},
```
Comment (symbolpunk) [inline website/static/get-started-faqs.json:20]:
```suggestion
{"question": "Can I withdraw my funds from Arbitrum back to Ethereum without going through the sequencer? What about funds that are in a contract?","answer": "<p>Yes, it is possible to permissionlessly send a message from Ethereum to be executed on Arbitrum, while bypassing the Sequencer. You can do this by using the DelayedInbox contract and force-including the message after a certain amount of time has passed (currently ~24 hours). You can find more information about this behavior <a href=\"https://docs.arbitrum.io/sequencer#unhappyuncommon-case-sequencer-isnt-doing-its-job\">here</a>.</p>\n\n<p>Keep in mind that you can execute any message in this way, be it a withdrawal of funds back to Ethereum, or a call to a contract.</p>\n\n<p>You can also find an example of force-inclusion in <a href=\"https://github.com/OffchainLabs/arbitrum-tutorials/tree/master/packages/delayedInbox-l2msg\">this tutorial</a>. </p>\n\n<p></p>\n\n","key": "can-i-withdraw-my-funds-from-arbitrum-back-to-ethereum-without-going-through-the-sequencer-what-about-funds-that-are-in-a-contract"},
```

non-blocking nit (FAQ CMS entry has been updated)

**Review by symbolpunk — APPROVED** [review]
LGTM! One non-blocking nit (applied to Notion already)

**Review by mahsamoosavi — APPROVED** [review]
LGTM :)

**Pull request merged**